content/blogs/newsletter-102.md

```diff
@@ -0,0 +1,131 @@
+as a whole isn't mentioned. Most of what we are going to talk about is the current situation with releases and overall ReactOS development.
+
+## Why does it take so long to release a new version of ReactOS? Are you guys dead?
+The latest release of ReactOS is 0.4.14, published on 16th of December, 2021. That release alone took a year to be released. Back then the ReactOS followed the 3-months cadence, which every 3 months
+a new release would be released to the public. But since 2021, ReactOS is still at 0.4.14. Are you guys dead?
+
+The answer is certainly no, the way we handle releases has changed. Back in the day the way ReactOS releases were made it was done for the sake of quantity than quality. Every new release after 3 months
+is similar to each other, with exceptions of a very few features and bug fixes shipped to the release. But overall the net gap between the releases wasn't that big.
+Now there is a rule that for a development release to reach "Release" status, it has to have a reasonable amount of regressions (no more than 20) and that the stability isn't badly impacted
+by the introduction of new features or code changes done during the development. In other words, the ReactOS project is now focusing on providing quality releases of which the 3-months cadence cannot
+be applied anymore. Such approach wasn't quite feasible for a development team size like ours anyway.
+
+So it is expected that it will take longer for a new release to be published, although such releases will now be substantial with lots of improvements than they were before.
+If you're impatient to test the latest changes and features you can always try the [nightly builds](https://reactos.org/getbuilds) (also called experimental builds).
+
+## Development status
+In the last year and during the beginning of 2023, the ReactOS developers and contributors alike are working on many fronts of the project, the top focused area being the kernel.
+Other areas that isn't kernel related are the applications, specifically our Paint and Notepad programs, the Input Method Editor (IME) as well as other stuff such as the ReactOS
+testcase infrastructure.
+
+## ReactOS x64 port status
+The x64 port of ReactOS, while still in its infancy, has steadily gotten better in terms of stability. Back then ReactOS could barely boot and if it ever did, it usually resulted
+in an immediate crash or a BSoD by just doing simple tasks. With the help of Timo Kreuzer (tkreuzer), the x64 port has gotten to a somewhat tolerable state where ReactOS can now
+operate without big problems.
+
+Although issues still do remain, and compatibility is inferior to the x86 version, more applications and even hardware drivers have started to work.
+Note that the x64 port currently will not run x86 applications due to lack of WoW64.
```

Comment:
```suggestion
Note that the x64 port currently will not run **ANY** x86 applications due to lack of WoW64.
This means that most applications cannot be installed, because generally installers are x86 applications.
```